hw/drivers/sensors/mpu6050/src/mpu6050.c

```diff
@@ -140,7 +136,7 @@ mpu6050_read8(struct sensor_itf *itf, uint8_t addr, uint8_t reg,
     rc = hal_i2c_master_write(itf->si_num, &data_struct,
                               OS_TICKS_PER_SEC / 10, 0);
     if (rc != 0) {
-        MPU6050_ERR("I2C access failed at address 0x%02X\n", addr);
+        MPU6050_ERR("I2C access failed at address 0x%02X\n", itf->si_add);
```

Comment:
"itf->si_add" should be "itf->si_addr" for all the log lines where address is logged.